types/bull-arena/index.d.ts

```diff
@@ -1,56 +1,80 @@
-// Type definitions for bull-arena 2.6
+// Type definitions for bull-arena 3.0
```

Comment:
This is the earliest version that these changes apply to; they also apply to 3.1 and 3.2, and there aren't any changes to the public API that correspond to those versions. Enough to just leave as 3.0, or will I need to submit requests for 3.1 and 3.2 as well?